static/i18n/en/app.yml

```diff
@@ -283,24 +283,26 @@ settings:
     about: Help
   display:
     desc:
-    language: Language
-    languageDesc: Choose the language to display.
+    language: Display language
+    languageDesc: Set the language displayed in Ledger Live.
     counterValue: Base currency
-    counterValueDesc: Choose the currency to display next to your balance and operations. An exchange is used (via BTC pair).
+    counterValueDesc: Choose the currency to display next to your balance and operations.
+    exchange: Rate provider ({{ticker}})
+    exchangeDesc: Choose the provider of the base currency exchange rates
     region: Region
-    regionDesc: Choose the region in which you’re located to set the application’s time zone.
+    regionDesc: Choose the region in which you’re located to set the Ledger Live's time zone.
     stock: Regional market indicator
     stockDesc: Choose Western to display an increase in market value in blue. Choose Eastern to display an increase in market value in red.
   currencies:
-    desc: Select a cryptocurrency to edit its settings.
+    desc: Select a cryptocurrency to edit its settings
     exchange: Rate provider ({{ticker}})
-    exchangeDesc: Choose the provider of the base currency exchange rates (via BTC).
+    exchangeDesc: Choose the provider of the base currency exchange rates
```

Comment:
this was a missing info i think we need to add. it's not a direct Crypto->Fiat but we always use the BTC as intermediary. `(via BTC)` . any other way to phrase it?

Reply:
This is an important topic which was already planned to be addressed. I'll revert, but it will change, possibly with some added disclaimers. I think we need a few extra tooltips or links to documentation to properly explain it.

Reply:
make sense